user-guide/settings/gateway.md

```diff
@@ -8,33 +8,43 @@ position: 30
 
 # Gateway Menu
 
-The __Gateway__ sub-menu provides options to easily configure how Fiddler Everywhere accesses the network. By default, Fiddler Everywhere "chains" to the system's default proxy. These settings allows you to overwrite that behavior.
+The __Gateway__ sub-menu provides options to easily configure how Fiddler Everywhere accesses the network. By default, Fiddler Everywhere "chains" to the system's default proxy. These settings allow you to overwrite that behavior.
 
 ![Example manual proxy configuration](../../images/settings/gateway-system-proxy.png)
 
 ## Use system proxy
 
-__Use system proxy (recommended)__ is the default selection. Fiddler uses the OS system proxy. Some browsers and many applications use the system proxy by default and re notified when it changes. 
+__Use system proxy (recommended)__ is the default selection. Fiddler uses the OS system proxy. Some browsers and many applications use the system proxy by default and are notified when it changes. 
 
 ## Manual proxy configuration
```

Comment:
There is a feedback comment that asks about proxy authentication. Is it supported, can we add a line about it?

Reply:
proxy authentication is not yet supported - I will add it once the team implements it